config/config.template.js

```diff
@@ -19,10 +19,11 @@ module.exports = {
     email: ''
   },
   tasks: {
-    "bootstrap-cdn": {minute: 3},
-    "google": {minute: 3},
-    "cdnjs": {minute: 3},
-    "jsdelivr": {minute: 3}
-    //jquery: {minute: 5}
+    // resync interval is in seconds
+    "bootstrap-cdn": {interval: 180},
```

Comment:
Why's that? there's not too many projects on bootstrap-cdn, I didn't think it was worth the effort